src/components/CollapsableItem.tsx

```diff
@@ -143,9 +144,10 @@ const StyledCollapsableContainer = styled.div<{ collapsed: boolean; deep: number
                 margin-left: -${collapseOffset}px;
             }
 
-            > ${StyledCollapsableItem}:before {
+            ${hasChild &&
+            `> ${StyledCollapsableItem}:not(:last-child):before {
```

Comment:
I'm not sure anybody can understand what is this

Reply:
add comment, will be refactored here https://github.com/taskany-inc/issues/issues/1448